docs/geedocs/5.2.5/answer/7160006.html

```diff
@@ -163,6 +163,11 @@ <h5>
           <th class="narrow">Description</th>
           <th>Workaround</th>
         </tr>
+        <tr>
+          <td>1158</td>
+          <td>Portable server loses geeServerDefs for a served map/globe as soon as it is viewed</td>
```

Comment:
```suggestion
          <td>Portable files served from Portable Server don't provide geeServerDefs to clients through the Maps API</td>
```

Reply:
@tst-jcupp this issue happens only if someone is viewing that map/globe. Otherwise if no one is viewing it, the issue does not happen and server definition is returned ok.

Reply:
So how do we want to word this?